client/src/Components/Spectrogram/SpectrogramPage.jsx

```diff
@@ -802,7 +807,18 @@ class SpectrogramPage extends Component {
 
         <div className="mt-3 mb-0 ml-0 mr-0 p-0" style={{ margin: '5px' }}>
           <Collapsible title="Annotations" style={{ marginTop: '5px', width: '100%' }}>
-            <Table columns={columns} data={this.calculateData(this.state.meta)} />
+            <Table
+              columns={[
+                { title: 'Annotation', dataIndex: 'annotation' },
```

Comment:
I'm not sure from a readability point of view I'm a fan of having the json for the columns here

Reply:
yeah i mean it just cant be at the very bottom of the file

Reply:
Thats fine, I would usually put them at the top of the file, just forgot to move it 😁